ss-tool.sh

```diff
@@ -240,7 +269,7 @@ if [ -n "$_configFile" ]; then
 	     	msg "FAILED:: $error_count errors in $i flyway logs";
 	     	msg "  View these by running 'docker logs compose-flyway-$i '"
 	     	
-	     	#TODO: below is a temp message whilst we iron out where schema creationg and public setup is
+	     	#TODO: below is a temp message whilst we iron out where schema creation 
```

Comment:
With the improvement you've made via `cleanSchemaCreate` has this TODO not been sorted out now? 
(ie: the lines for message output of msg `"NOTE:: if this is the first ` etc. could also be removed couldn't they?)

Reply:
Thanks
Removed comment here https://github.com/Bank-Builder/ss-tool/pull/5/commits/227e18d735bec8735f192dd8d4709d9f1b9b0d8d